conf.yaml

```diff
@@ -1665,8 +1666,8 @@ contents:
           - title:      Integrations Developer Guide
             prefix:     en/integrations-developer
             current:    master
-            branches:   [ master ]
-            live:       [ master ]
+            branches:   [ {main: master} ]
+            live:       [ main ]
```

Comment:
You can just remove `live`. If not present, the full list of `branches` will be considered "live" (and accessible to search engines, etc.)